### ISSUE TITLE
jaxrs-cxf-client - Generate error in @ApiResponse when the response class is not specified

### ISSUE THREAD

**Issue by liberoguerra** [swagger-api/swagger-codegen#7853]
##### Description
ISSUE:
I use language **jaxrs-cxf-client** and inside json file there is not specified a response class for HTTP Status code 401.
After generation, the api class contains several errors about @ApiResponse. In detail the plugin generates this line in error "**response = .class**"

API class generated (Java code)
```java
...
	@ApiResponses(value = {
		@ApiResponse(code = 200, message = "OK", response = ResponseBody.class),
		@ApiResponse(code = 401, message = "Unauthorized", response = .class)
	})
...
```

##### Swagger-codegen version
2.2.3 and 2.3.1

##### Swagger declaration file content

```json
...
	"responses":{
		"200":{
			"description":"OK",
			"schema":{
				"$ref":"#/definitions/ResponseBody"
			}
		},
		"401":{
			"description":"Unauthorized"
		}
	}
...
```

##### Command line used for generation

I use maven generate goals with this configuration:
```xml
<configuration>
	<inputSpec>scr/main/resources/file.json</inputSpec>
	<language>jaxrs-cxf-client</language>
	<configOptions>
		<sourceFolder>codegen</sourceFolder>
		<apiPackage>com.example.rest.api</apiPackage>
		<modelPackage>com.example.rest.model</modelPackage>
		<generateApiTests>false</generateApiTests>
		<generateModelTests>false</generateModelTests>
		<useJaxbAnnotations>true</useJaxbAnnotations>
	
	</configOptions>
</configuration>
```

**Comment by jmini** [comment]
Thank you for this report. I have proposed a fixed for the 2.x version here: #7886.

I have taken this opportunity to work on a port of `jaxrs-cxf-client` generator for v3 (I will link the corresponding PR here).

**Comment by jmini** [comment]
Fix for v3: https://github.com/swagger-api/swagger-codegen-generators/pull/47

**Comment by jmini** [comment]
In v3, the `swagger-codegen-generators` jar now contains the fix.

The next `swagger-codegen` build will produce a `cli` jar that contains the lattest version of `swagger-codegen-generators`. You can also build it locally as I did to run `bin/jaxrs-cxf-client-petstore.sh` which demonstrates that everything works as expected: see #7904

You are writting that you are using maven. I guess that if you run your build with `-U`, using version `3.0.0-SNAPSHOT` of the `swagger-codegen-maven-plugin` maven will also fetch `io.swagger:swagger-codegen-generators:1.0.0-SNAPSHOT` 

You can give it a try and report feedback here.

**Comment by jmini** [comment]
@HugoMario @webron, can you please add `cxf-client` and `issue` labels? Depending on if #7886 gets merged or not the version will be `2.4.0` or `3.0.0`

**Comment by jmini** [comment]
Both PR have been fixed for v2 and v3. 
@webron, @HugoMario or someone else, can you add "jaxrs-cxf-client" and "issue" Labels? Milestone is "2.4.0".

This issue can be closed.
